src/quo/components/wallet/swap_input/view.cljs

```diff
@@ -0,0 +1,118 @@
+(ns quo.components.wallet.swap-input.view
+  (:require [quo.components.avatars.token-avatar.view :as token-avatar]
+            [quo.components.buttons.button.view :as buttons]
+            [quo.components.dividers.divider-line.view :as divider-line]
+            [quo.components.markdown.text :as text]
+            [quo.components.tags.network-tags.view :as network-tag]
+            [quo.components.wallet.approval-label.schema :as approval-label.schema]
+            [quo.components.wallet.approval-label.view :as approval-label]
+            [quo.components.wallet.swap-input.style :as style]
+            [quo.foundations.colors :as colors]
+            quo.theme
+            [react-native.core :as rn]
+            [schema.core :as schema]))
+
+(def ?schema
```

Comment:
I think it's good, that we move the schema to `quo.components.wallet.swap-input.schema` ns. WDYT?

Reply:
IMO they should live next to the component instead of having its own namespace.

@ilmotta do we have an agreement on this or it's still open to discussion?

Reply:
This point is open for discussion. Some time ago I added the following to this meta issue https://github.com/status-im/status-mobile/issues/18726:

```
[2024-05-21] Should quo schemas be always defined in a separate file?

Components can wildly vary in their API (their props). Some are complex
due to complexity in the Design System and use malli's multidispatch
feature. But the majority is simpler.

By always writing the component schema in a separate file schema.cljs we
create an indirection for something that's very core the component API,
not like the file style.cljs, which is often tangential to the
understanding of the component. This is arguably not the best DX for
simple schemas.

So, should we always put quo schemas in separate schema.cljs files? Or
should we do it only when the readability of the schema affects the
component.cljs file due to its size/complexity?
```

Reply:
I think there are clear cases where a separate file is useful, in components with more complex schemas. But other times it just seems a bit excessive to always have a separate namespace. Sometimes it feels to me a bit like moving the function's docstring out of the namespace, which would be weird of course. On the other hand, it's nice to have schemas in separate files in terms of finding them directly in the editor.

I don't have a strong opinion on this, but I wish we could keep the rule more flexible.

Reply:
Thanks @ilmotta  I agree with the flexibility